test/commands/extended/get_bundles_test.py

```diff
@@ -92,29 +103,43 @@ def test_fail_unexpected_parameters(self):
 
     def test_fail_transaction_wrong_type(self):
         """
-        ``transaction`` is not a TrytesCompatible value.
+        ``transactions`` contains no TrytesCompatible value.
         """
         self.assertFilterErrors(
             {
-                'transaction': 42,
+                'transactions': [42],
             },
 
             {
-                'transaction': [f.Type.CODE_WRONG_TYPE],
+                'transactions.0': [f.Type.CODE_WRONG_TYPE],
             },
         )
 
     def test_fail_transaction_not_trytes(self):
         """
-        ``transaction`` contains invalid characters.
+        ``transactions`` contains invalid characters.
         """
         self.assertFilterErrors(
             {
-                'transaction': b'not valid; must contain only uppercase and "9"',
+                'transactions': [b'not valid; must contain only uppercase and "9"'],
             },
 
             {
-                'transaction': [Trytes.CODE_NOT_TRYTES],
+                'transactions.0': [Trytes.CODE_NOT_TRYTES],
+            },
+        )
+
+    def test_fail_no_list(self):
+        """
+        ``transactions`` has one hash rather than a list of hashes.
+        """
+        self.assertFilterErrors(
+            {
+                'transactions': self.transactions[0],
+            },
+
+            {
+                'transactions': [f.Type.CODE_WRONG_TYPE],
```

Comment:
❔ Could we also add tests for invalid values inside of `transactions`?  See `test.commands.extended.send_trytes_test.SendTrytesRequestFilterTestCase.test_fail_trytes_contents_invalid` for an example.